project-reports/2023/2023-Q2-Hyperledger-Cacti.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+title: 2023 Q2 Hyperledger Cacti
+parent: 2023
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# 2023 Q2 Hyperledger Cacti
+
+## Project Health
+
+- The first code merge of Cactus and Weaver to "Cacti" repository was completed and we released Cacti v1.2.0 in the end of March.
+- The maintainers are also discussing the [new architecture of Cacti](https://github.com/hyperledger/cacti/blob/main/ROADMAP.md#cacti-v2) toward Cacti-V2 in order to combine the strengths of the original Cactus and Weaver systems to provide a more flexible interoperablity tool with several feature selection options given to network administrators.  The Cacti v2-alpha, v2-beta and v2.0 release will be redesigned according to the new architecture.
+
+# Questions/Issues for the TSC
+
+- None at the moment.
+
+# Releases
+
+- v1.2.0 => https://github.com/hyperledger/cacti/tree/v1.2.0
+    - including the features: merging Weaver into Cacti
+
+# Overall Activity in the Past Quarter
+
+1. Major update as merged pull-requests
+    - merging from Cactus and Weaver:
+        - [new_feature] Merge Weaver to Cacti
+            - https://github.com/hyperledger/cacti/commit/4ac560a3222733a2ecf465f80f42b2007ddcb6be
+            - https://github.com/hyperledger/cacti/commit/4171eebd83aba627a155509e13bd95fb50f6ae73
+    - support tools:
+        - [new_feature (WIP)] Cacti GUI app 
+            - https://github.com/hyperledger/cacti/pull/2265
+2. Created a new architecture diagram toward realizing Cacti as a more flexible interoperability toolkit
```

Comment:
Can you provide a link to the architecture diagram?

Reply:
@takeutak Can you add two links: one to the position in `ROADMAP.md` where the architecture diagram is displayed (https://github.com/hyperledger/cacti/blob/main/ROADMAP.md#cacti-v2), and another to the image file (https://github.com/hyperledger/cacti/blob/main/images/cacti-architecture-v1-merge.png)?

Reply:
@tkuhrt @VRamakrishna added https://github.com/hyperledger/toc/pull/92/commits/1f331d8fd11b9a74c0a030aad6e82e874a1344f7